includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -531,6 +531,16 @@ public function should_show_payment_request_button() {
 			return false;
 		}
 
+		// Cart total is 0 or is on prouct page and product price is 0.
```

Comment:
Nitpick: There is a typo in "product page".

Reply:
fixed in [b2e4c9b](https://github.com/Automattic/woocommerce-payments/pull/6334/commits/b2e4c9b0b25311c882f56a142b867916a64d6880) . thx .